app/controllers/admin/contact_translations_controller.rb

```diff
@@ -2,16 +2,17 @@ class Admin::ContactTranslationsController < Admin::BaseController
   include TranslationControllerConcern
   layout :get_layout
 
-  def index
-  end
+  def index; end
+
   def edit
     render_design_system(:edit, :legacy_edit)
   end
+
 private
 
   def get_layout
-    design_system_actions = %w[index confirm_destroy]
-    design_system_actions += %w[edit] if preview_design_system?(next_release: false)
+    design_system_actions = %w[confirm_destroy]
+    design_system_actions += %w[edit index] if preview_design_system?(next_release: false)
```

Comment:
i think this was right before wasn't it? The index view is only ever in the GOV.UK Design System?

Reply:
ya but there was some issue while running the controller test so I changed it